code/lists/ckey_configs.dm

```diff
@@ -0,0 +1,104 @@
+// Various configuration lists full of player ckeys
+
+// Some of these are loaded in world/New(), some are loaded here.
+// not sure why and don't want to risk breaking more things
+
+var/global
+	// Admins ( ["ckey"] = "rank" )
+	// Populated by proc call in world.New()
+	list/admins = list()
+	list/onlineAdmins = list()
+
+	// HoS/NTSO-whitelisted players
+	list/NT = load_config_list("config/nt.txt")
+
+	// Mentors
+	list/mentors = load_config_list("config/mentors.txt")
+
+	// Players whomst'd've get allowed if whitelist-only is enabled
+	list/whitelistCkeys = list()
+
+	// Players who are allowed to bypass the server's player cap
+	list/bypassCapCkeys = list()
+
+	// Hellbanned players
+	// (see code/hellban.dm)
+	list/hellbans = load_config_list("config/hellbans.txt")
+
+
+
+
+
+/**
+* Gets a list of ckeys from a file, ignoring comments/blank lines
+* This could probably be refactored to config files in general, but
+*/
+/proc/load_config_list(filename, as_ckey = 1, secrets = 0)
+	// "fexists doesn't count symlinks as files"
+	// if (!fexists(filename)))
+	//		throw EXCEPTION("lummox!!!!!!!!!!!!!!")
+
+	var/list/result = list()
+
+#ifdef SECRETS_ENABLED
+	if (!secrets)
+		// If secrets are enabled, load config options from +secrets as well.
+		// This way you can still have local additions to it.
+		// This can probably be improved, since you can't *remove* entries,
+		// just add new ones...
+		result += load_config_list("[CONFIG_PATH_PREFIX]filename", as_ckey, 1)
+#endif
+
+	try
+		// Get every line in the file as a list of lines
+		// [CONFIG_PATH_PREFIX] here reads from +secret/config/ instead of config/,
+		// if secrets are enabled.
+		var/list/lines = dd_file2list(filename)
+
+		// Turn the list of lines into a different list of *valid* lines
+		for(var/line in lines)
+			if (!line)
+				// Blank lines don't count
+				continue
+			if (copytext(line, 1, 2) == "#")
+				// Comments don't count either
+				continue
+
+			result += (as_ckey ? ckey(line) : line)
+
+	catch (var/exception/e)
+		// this might not ever happen depending on what dd_file2list does
+		// in the event of missing files. oh well
+		logDiary("Failed to load config file '[filename]': [e]\n")
```

Comment:
```suggestion
		logDiary("Failed to load config file '[filename]': [e]\n")
		stack_trace("Failed to load config file '[filename]': [e]")
```
Diary is pretty hidden, this is a fairly important thing that could crash and having it actually be in view-runtimes might help noticing it.